### ISSUE TITLE
Use description as alt text if no title when dropping on thumbnail

### ISSUE THREAD

**Issue by theefer** [guardian/grid#176]
Dropping an image on the trail doesn't get you any text to edit down as alt text if the title is missing. It should probably use the description instead in that case.

Description is usually mapped to caption, but there's no caption for thumbnail so the copy is not passed through.
